src/MeetingsSDKAdapter.js

```diff
@@ -639,21 +662,36 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
     const muted = {
       ID: VIDEO_CONTROL,
       icon: 'camera-muted_28',
-      tooltip: 'Start video',
+      tooltip: 'Start Video',
```

Comment:
leave them as they were, if needed, @lalli-flores will ask for updating these labels in a separate story